code/Migration.php

```diff
@@ -14,4 +14,189 @@
+		$queried = false;
+		$existingColumns = self::getTableColumns($table);
+		if ($existingColumns) {
+			foreach ($columns as $column => $properties) {
+				if (!array_key_exists($column, $existingColumns)) {
+					DB::query("ALTER TABLE $table ADD $column $properties;");
+					$queried = true;
+				}
+			}
+		}
+		return $queried;
+	}
+
+	/**
+	 * Gets the value for a single column in a row from the database by the ID column.
+	 * Useful when a field has been removed from the class' `$db` property,
+	 * and therefore is no longer accessible through the ORM.
+	 * Returns `null` if the table, column or row does not exist.
+	 *
+	 * @param string $table
+	 * @param string $columns
+	 * @param string||int $id
+	 * @return string
+	 */
+	protected static function getRowValueFromTable($table, $field, $id) {
+		$value = null;
+		if (self::tableColumnExists($table, $field)) {
+			$query = new SQLQuery();
+			$query->setFrom($table)->setSelect([$field])->setWhere("ID = $id");
+			$results = $query->execute();
+			if ($results) {
+				foreach ($results as $result) {
+					$value = $result[$field];
+					break;
+				}
+			}
+		}
+		return $value;
+	}
+
+	/**
+	 * Gets the values for multiple rows on a database table by the ID column.
+	 * Useful when fields have been removed from the class' `$db` property,
+	 * and therefore are no longer accessible through the ORM.
+	 * Returns an empty array if the table, any of the columns or the row do not exist.
+	 *
+	 * @param string $table
+	 * @param array $columns
+	 * @param string||int $id
+	 * @return array ['FieldName' => value]
+	 */
+	protected static function getRowValuesFromTable($table, array $fields, $id) {
+		$values = [];
+		if (self::tableColumnsExist($table, $fields)) {
+			$query = new SQLQuery();
+			$query->setFrom($table)->setSelect($fields)->setWhere("ID = $id");
+			$results = $query->execute();
+			if ($results) {
+				foreach ($results as $result) {
+					foreach ($fields as $field) {
+						$values[$field] = $result[$field];
+					}
+					break;
+				}
+			}
+		}
+		return $values;
+	}
+
+	/**
+	 * Sets the values for multiple rows on a database table by the ID column.
+	 * Useful when fields have been removed from the class' `$db` property,
+	 * and therefore are no longer accessible through the ORM.
+	 * Returns false if the table or any of the rows do not exist.
+	 * Returns true if the SQL query was executed.
+	 *
+	 * @param string $table
+	 * @param array $values ['FieldName' => value]
+	 * @param string||int $id
+	 * @return boolean
+	 */
+	protected static function setRowValuesOnTable($table, array $values, $id) {
+		$queried = false;
+		if (self::tableColumnsExist($table, array_keys($values))) {
+			$query = "UPDATE $table SET";
+			$valuesCount = count($values);
+			$i = 0;
+			foreach ($values as $field => $value) {
+				if (is_string($value)) $value = "'" . Convert::raw2sql($value) . "'";
+				$query .= " $field = " . $value;
```

Comment:
`$table` and `$field` should be escaped (too easy to have MySQL keywords as column names). Tip: Also you can just build an array and join it to generate the comma separated values, if you wanted (end result is the same so don't change it if you don't want to).